packages/workbox-core/index.mjs

```diff
@@ -18,7 +17,6 @@ class WorkboxCore {
   }
 }
 
-export {logger};
-export {WorkboxError};
+export {_private};
```

Comment:
Can this be written as `export _private;`, since `_private` is already an object with the named exports as keys? Or does it have to be wrapped in another `{}`?

Reply:
Has to be wrapped.